sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/SortBenchmark.scala

```diff
@@ -80,7 +80,7 @@ class SortBenchmark extends BenchmarkBase {
       }
       val buf = new LongArray(MemoryBlock.fromLongArray(array))
       timer.startTiming()
-      RadixSort.sort(buf, size, 0, 7, false, false)
+      RadixSort.sort(buf, size.toLong, 0, 7, false, false)
```

Comment:
As I wrote before, I wanted to emphasis the difference of types between caller and callee.
It is OK to reply on implicit cast, done.